app/views/facility_reservations/_reservation_fields.html.haml

```diff
@@ -1,4 +1,8 @@
 - if @reservation.admin_editable?
+  - if @reservation.admin?
+    .control-group
+      = f.input :category do
+        = f.select :category, options_for_select(Reservation::CATEGORIES.map {|c| [t("reservations.categories.#{c}"), c]}, f.object.category), include_blank: "Select a Category"
```

Comment:
See `FacilityReservationsHelper#offline_category_collection`. That might allow you to clean this up a little and keep things consistent.